### DISCUSSION TITLE
收到OpenAI API的邮件：Access Terminated

### DISCUSSION THREAD

**Discussion by greatlsd** [ChatGPTNextWeb/NextChat#2747] [General]
我确信没干啥特别的操作，只是一周以来在vercel上一键部署了3、4次，再就是自己的linode的美国服务器上部署了，一共用了1周，只在家里和办公室用过，除了问问ubuntu的命令，或者发个“你好”进行测试，再没发过什么问题。今天中午刚发给一个异地的朋友测试，并且发现版本从2.9.4到2.9.5有更新，然后就顺手在linode上也做了更新，然后很快就收到邮件禁用了，也不知道哪个行为触发了条例。当时一周前一口气充了20美元进去，早上刚看过，一周来才花了0.46美元，然后现在莫名其妙啥都没了，连chat.openai.com界面都用不了，提示账号deactivated

大家有遇到这种情况吗？如何申诉？

邮件内容如下：
Hi there,

After a thorough investigation, we have determined that you or a member of your organization are using the OpenAI API in ways that violate our policies.

Due to this breach we are halting access to the API immediately for the organization Personal. Common reasons for breach include violations of our usage policies or accessing the API from an unsupported location. You may also wish to review our Terms of Use.

If you believe this is in error and would like to appeal, please contact us through our help center. We will review appeals as soon as possible and will contact you if we reinstate access to the API.

**Comment by iotwq** [comment]
兄弟解决了没，我也遇到了，fuck openai

**Comment by Wznnnnn** [comment]
同申诉中，太无语了，(第一天)

错误如下：
We have determined that you are using OpenAI services in ways that violate our policies.

Due to this breach we are halting access to our products immediately for the account associated with the email    xxxxxxxxxxxxxxxxxx. Common reasons for breach include violations of our usage policies or accessing our services from an unsupported locations. You may also wish to review our Terms of Use 
 
If you believe this is in error and would like to appeal, please contact us through our help center.We will review your appeal and will contact you if we reinstate your access.

Best,
The OpenAI team.

**Comment by Wznnnnn** [comment]
> 大家有遇到这种情况吗？如何申诉？

写邮件给support@openai.com，然后回信中会有help center的链接，点进去在客服里输入Appeal之类的内容，会有机器人引导，说是3天会回复

**Reply by fengshgx** [reply]
后续怎样了？
